torch/_dynamo/variables/builtin.py

```diff
@@ -1120,8 +1120,7 @@ def call_getattr(
                     ):
```

Comment:
@ysiraichi do you remember why you guarded the logic above to match if bases is object or TensorBase? @Fidget-Spinner's code below actually now subsumes this case above but I feel there must have been some reason you wrote it in a restricted way rather than the more generic way below

Reply:
I was just trying to be restrictive of the types that went through this branch, since we have no sources to generate a guard for. I guess the question is whether we need to worry about sourceless types in `type.__bases__`.